src/validators/arguments.rs

```diff
@@ -117,17 +141,31 @@ impl BuildValidator for ArgumentsValidator {
             });
         }
 
+        let py_var_kwargs_mode: Bound<PyString> = match schema.get_as(intern!(py, "var_kwargs_mode"))? {
+            Some(v) => v,
+            None => PyString::new_bound(py, "single"),
+        };
+        let var_kwargs_mode = VarKwargsMode::from_str(py_var_kwargs_mode.to_string().as_str())?;
+        let var_kwargs_validator = match schema.get_item(intern!(py, "var_kwargs_schema"))? {
+            Some(v) => Some(Box::new(build_validator(&v, config, definitions)?)),
+            None => None,
+        };
```

Comment:
```suggestion
        let var_kwargs_validator = schema
            .get_item(intern!(py, "var_kwargs_schema"))?
            .map(|v| Box::new(build_validator(&v, config, definitions)));
```

Just some suggestions to help avoid unnecessary arms on match statements :)

Reply:
`var_kwargs_validator` is now of type `Option<Box<Result<CombinedValidator, PyErr>>>
`, and adding the `?` operator to `build_validator` raises the following error:

```
the `?` operator can only be used in a closure that returns `Result` or `Option` (or another type that implements `FromResidual`)
the trait `FromResidual<std::result::Result<Infallible, pyo3::PyErr>>` is not implemented for `Box<CombinedValidator>`
```

Not sure what this is about.

Reply:
The option-of-result nesting here makes it tricky, I think the original `match` was probably fine.